src/lib/jsonParseV2/jsonRequestTreat.cpp

```diff
@@ -46,7 +46,7 @@
 *
 * jsonRequestTreat - 
 */
-std::string jsonRequestTreat(ConnectionInfo* ciP, ParseData* parseDataP, RequestType requestType, JsonDelayedRelease* releaseP)
+std::string jsonRequestTreat(ConnectionInfo* ciP, ParseData* parseDataP, RequestType requestType, JsonDelayedRelease* releaseP, std::vector<std::string>& compV)
```

Comment:
I did it in the header file (the line is 8 chars longer :-)) so yes, I will do it in the source file as well.